### PR TITLE
add nestjs with express/fastify for benchmarker

### PR DESCRIPTION
**Pull request by zieglar**
Closes [#954](https://github.com/the-benchmarker/web-frameworks/issues/954#issuecomment-520960790)

### CODE REVIEW

File: javascript/nestjs-fastify/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "@nestjs/common": "^6.10.14",
```
Comment (waghanza) [inline javascript/nestjs-fastify/package.json:3]: could you use `~6.10.14` instead of `^6.10.14` ?

**Comment by zieglar** [comment]
> I assume some files are build by `nest`. How are they generated ?
> 
> It is better to generate them directly onto the container

if want generate them directly onto the container, javascript Dockerfile must rebuild :(

**Comment by waghanza** [comment]
Yep, this is the idea.

But I do now know which `nest` command to use

**Comment by zieglar** [comment]
> Yep, this is the idea.
> 
> But I do now know which `nest` command to use

If you want, I can submit source code.
step list:
1. run `npm -g install @nestjs/cli`
2. run `nest build`
3. copy dist/[*.ts,*.js,*.map] to container 
4. copy package.json to container
5. run `npm install`
6. pm2 start app.js

**Comment by waghanza** [comment]
> If you want, I can submit source code.

It will be awesome

**Comment by waghanza** [comment]
I will create an other `PR` based on your code, to give you my corrections after that :stuck_out_tongue:

**Comment by zieglar** [comment]
> I will create an other `PR` based on your code, to give you my corrections after that 😛

done

**Comment by waghanza** [comment]
I have an error while running `nest build`
~~~javascript
node_modules/@nestjs/common/interfaces/external/mqtt-options.interface.d.ts:104:39 - error TS2580: Cannot find name 'Buffer'. Do you need to install type definitions for node? Try `npm i @types/node`.

104     ca?: string | string[] | Buffer | Buffer[];
                                          ~~~~~~
~~~
This, however, seems to be a warning, but fails docker creation

**Comment by zieglar** [comment]
> I have an error while running `nest build`

maybe run `npm install` before run `nest build` ?

**Comment by waghanza** [comment]
That's what I do ...

**Comment by zieglar** [comment]
> That's what I do ...

I update `package.json` and now nest build is ok

**Comment by waghanza** [comment]
@zieglar results has been updated

**Pull request closed without merging**